ts/components/dialog/OnionStatusPathDialog.tsx

```diff
@@ -84,26 +98,45 @@ const OnionPathModalInner = () => {
                 );
               })}
             </Flex>
-          </div>
+          </StyledLightsContainer>
           <Flex container={true} flexDirection="column" alignItems="flex-start">
-            {nodes.map((snode: Snode | any, index: number) => {
+            {nodes.map((snode: Snode | any) => {
```

Comment:
if we are using map shouldn't the react props have a key property? https://reactjs.org/docs/lists-and-keys.html

Reply:
the key was actually inserted in the element returned a few lines below in `OnionCountryDisplay`, but yeah I moved the setting of the key here cause it makes more sense.